### ISSUE TITLE
Clarification sought on "set of web pages"

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2298]
The definition of a set of web pages indicates a collection of web pages with a common purpose - it would be helpful to have more examples beyond the publication example give.  

For example - within a financial institution - are the web pages related to auto loans considered as part of the same set as pages on home loans?  

With use of this term in the Findable Help SC it means we need to have more consistency in how we collectively agree it should be understood.

**Comment by fstrr** [comment]
What about:
1. A jobs / careers section of a web site. All of the pages relating to current vacancies, what it's like to work for the company, etc. would be a set of pages with a common purpose. One thing that would need clarifying would be whether a vacancy listing on the employer's site is in the same set of pages as the pages that handle applying for the job if the application process is handled by a third-party product.
2. A news / press room with news releases.
3. For a software company: each product has a set of pages where users can read about the about the application, download a trial version, contact support, read release notes, etc. The collection of products is a set of web pages and the pages for each product is a nested set of web pages.

**Comment by mraccess77** [comment]
@fstrr the collections of products being a set of pages is something that I'm unclear on.  If the product pages are different from other product pages but created by the same group (marketing) and have the same purpose to "sell products" are all products really a set or just pages for a particular product or suite a set of pages?

**Comment by mbgower** [comment]
There has always been a somewhat self-fulfilling quality to the definitions of Set of Pages, where basically some pages that share a whole bunch of the same features of navigation, etc., become a Set of Pages as a result of sharing those features.

> The publication is logically a single contiguous unit, and contains navigation features that enable access to the full set of pages.

Essentially, a Set of Pages comes into existence by sharing the qualities on which we judge them. So for any of the previously listed scenarios (or for that matter for any page within those scenarios) Set of Pages may cease to be appropriate when the shared qualities aren't shared anymore.

This is especially the case as other SCs that rely on Set of Pages add-on more qualifiers for what constitutes the set.  So, for Consistent Help, a subset of the set of pages is identified that share the same help qualities.

> If a [web page](https://w3c.github.io/wcag/guidelines/22/#dfn-web-page-s) contains any of the following help mechanisms, and those mechanisms are repeated on multiple web pages within a [set of web pages](https://w3c.github.io/wcag/guidelines/22/#dfn-set-of-web-pages)

In other words, the Set of Pages takes on its context and 'requirements' based on the SCs in which it is evoked. And the set of pages is not consistent across the five SCs that invoke it, even within the same application, IMO.

**Comment by fstrr** [comment]
@mraccess77 Thinking about this some more, I would say that a user would most likely expect that buying something from a site should have a consistent set of navigation across all product pages.

It reminds me of working on information architecture for intranets: many intranets grow organically and mirror the structure of the business, which isn't helpful for task completion. For example: as an employee, I don't care which deeply-nested team deals with refunding expense reports, I just want to deal with a refund on my expense report without knowing the org chart for the finance department. Similarly, I don't care if your website has separate divisions for Windows, Mac, and Linux software, it's all just software and I want to find the help information.

**Comment by mraccess77** [comment]
I agree it would be helpful to have all product pages across products have consistent ways to find help - but I'm not sure that's required under the definition.  It seems like the person making the conformance claim should really define what is within a set of web pages it seems the choices are
* How you interpret common purpose
* If you choose author, group, or organization as the creator.

**Comment by patrickhlauke** [comment]
> There has always been a somewhat self-fulfilling quality to the definitions of Set of Pages, where basically some pages that share a whole bunch of the same features of navigation, etc., become a Set of Pages as a result of sharing those features.

indeed...

**Comment by bruce-usab** [comment]
> I agree it would be helpful to have all product pages across products have consistent ways to find help - but I'm not sure that's required under the definition.

I agree that it is not explicitly required under the definition.  For example, one could have a single site renting movies and selling home goods.  The branding and integration between those experiences are probably superficial.

> It seems like the person making the conformance claim should really define what is within a set of web pages...

Agreed and I think that is okay, as it is also going to be the case that any audit would need to define what in scope (including within a set of web pages),
